### PR TITLE
Remove older irida.cart.js angularjs service

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Sorry about the formatting changes on pipeline.launch.js

This remove the very old angularjs cart service and replaces it with functions that are already existing within the front end cart api.

## Related issue
Related to #477 

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [X] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
~~* [ ] Tests added (or description of how to test) for any new features.~~
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

File: src/main/webapp/resources/js/modules/cart/irida.cart.js [hunk 1]
```diff
@@ -1,37 +0,0 @@
-import angular from "angular";
-import { CART } from "../../utilities/events-utilities";
-import { emptyCart, removeSample } from "../../apis/cart/cart";
-
-function CartService(scope, $http) {
```
Comment (tom114) [inline src/main/webapp/resources/js/modules/cart/irida.cart.js:5]: I see the `removeSample` function reimplemented, what about the other service functions in here?  I'm assuming they're already done somewhere else?

File: src/main/webapp/resources/js/pages/pipelines/pipeline.launch.js [hunk 2]
```diff
@@ -307,8 +307,12 @@ function PipelineController(
    * @param projectId the project id of the sample to remove
    * @param sampleId the sample if to remove
    */
-  vm.removeSample = function(projectId, sampleId) {
-    CartService.removeSample(projectId, sampleId).then(function() {
+  vm.removeSample = function (projectId, sampleId) {
```
Comment (tom114) [inline src/main/webapp/resources/js/pages/pipelines/pipeline.launch.js:310]: So I'm assuming this is just changing to the cart functions in `resources/js/apis/cart/cart.js`?

Comment (joshsadam) [reply src/main/webapp/resources/js/pages/pipelines/pipeline.launch.js:310]: Yes that is all that is happening here.

File: src/main/webapp/resources/js/modules/cart/irida.cart.js (hunk 1 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/modules/cart/irida.cart.js:5]: So the answer to all your questions is that these functions had already been re-written for updates to other features (mostly the new cart).  The only functionality that was really still used was remove sample from cart I just refactored to use the new one.

**Pull request merged**